src/components/Onboarding/Onboarding.js

```diff
@@ -19,15 +18,31 @@ type Props = {
   onClose: () => void,
   headerMarkdown: string,
   logoURL: string,
+  analyticsShown: boolean,
+  analyticsAllowed: boolean,
+  analyticsAllowedChanged: (value: boolean) => void,
 };
 
 function Onboarding(props: Props) {
-  const { logoURL, headerMarkdown } = props;
+  const {
+    logoURL,
+    headerMarkdown,
+    analyticsShown,
+    analyticsAllowed,
+    analyticsAllowedChanged,
+  } = props;
   // translator: Shown on the onboarding screen. To visit it, open Wheelmap in an incognito window.
   const unknownAccessibilityIncentiveText = t`Help out by marking places!`;
   // translator: Button caption shown on the onboarding screen. To visit it, open Wheelmap in an incognito window.
   const startButtonCaption = t`Okay, let’s go!`;
 
+  // translator: Cookie notice with link to privacy policy
+  const cookieNotice = t`We use cookies to create a better and unique user experience. For more information about use and how to disable it, please see our <a href="https://news.wheelmap.org/terms-of-use/" target="_blank">Privacy Policy</a>. By using this website, you consent to the use of cookies.`;
+  // translator: Analytics checkbox
+  const analyticsCheckboxLabel = t`Share analytics data`;
+  // translator: Text asking for opt-in to analytics
+  const analyticsDescription = t`We use google analytics to improve your experience continously. Your decision to share data with google analytics can be revoked any time.`;
```

Comment:
-> `We use Google Analytics to improve your experience continuously. Your decision to share data with Google Analytics can be revoked any time.`

Reply:
Fixed

Reply:
Sorry, another take because I didn't get that there was much more text in the new feature :)

How about removing the second paragraph in the UI and use `Share analytics data with Google` as checkbox label?

Reply:
Sounds good to me.